src/Input/index.js

```diff
@@ -41,6 +43,15 @@ const Input = ({
     search ? "search" : "",
   ].join(" ");
 
+  const endIconJsx = displayClearInputIcon ? (
+    <div
```

Comment:
Let's add some props for accessibility to this:

```suggestion
    <div
       role="button"
       tabIndex="0"
       aria-label="Clear Input"
```